codegen/src/generator/rest_json.rs

```diff
@@ -115,6 +115,18 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+fn http_code_to_status_code(code: Option<i32>) -> Option<String> {
+    match code {
+        Some(actual_code) => {
+            match actual_code {
+                200 => Some("StatusCode::Ok".to_string()),
+                _ => None,
```

Comment:
I think there's a constructor for the status codes from numbers and I plan on wrapping that in here.